packages/nuxt-locokit/src/pages/auth/LostPassword.vue

```diff
@@ -61,6 +62,8 @@ const sendResetPasswordLink = async (data: string) => {
   formSentAndValid.value = !error.value
 }
 
+definePageMeta({ middleware: ['anonymous-routes'] })
```

Comment:
Mmm, just to be sure, you [ have to / must to ] define the middleware `anonymous-route` to redirect the user ?

Why don't you define this globally thanks to the meta information at the route level ?

That was - I think - the reason of a global middleware ?

I'm not sure it's a good way to do this at the component level, as we could forget to add this ?

Reply:
Fine, as it wasn't the initial way, I was asking myself.
Ok :-)